From 19e855143af95533541a07f8692a41c680085790 Mon Sep 17 00:00:00 2001
From: port-worker <port-worker@localhost>
Date: Tue, 11 Aug 2026 08:49:33 +0000
Subject: [PATCH] vendor_lineage: Ignore neverallows... again

Because unofficial builds are better than no builds!

[ported to A16: FRAMEWORK_LINEAGE_PLATFORM_API_FILE block no longer exists in
build/core/config.mk; inserted before the QCOM rules include instead.]

Change-Id: I4b19d09b28f79c6f5020bf03fdf8931145bca03a
---
 build/core/config.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build/core/config.mk b/build/core/config.mk
index 2359371d..a5ceb21d 100644
--- a/build/core/config.mk
+++ b/build/core/config.mk
@@ -13,5 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# We modify several neverallows, so let the build proceed
+ifneq ($(TARGET_BUILD_VARIANT),user)
+SELINUX_IGNORE_NEVERALLOWS := true
+endif
+
 # Rules for QCOM targets
 include $(TOPDIR)vendor/lineage/build/core/qcom_target.mk
-- 
2.43.0

